.github/workflows/issue_type_predicter.yaml

```diff
@@ -0,0 +1,89 @@
+          string: ${{ github.event.issue.body }}
+          replace-with: '-'
+
+       #Checks for missing information inside the issue content
+      - name: Check Information
+        id: check-info
+        run: |
+          ls -la
+          echo "analysis_response=$(pwsh .\\.github\\scripts\\issue_analyzer.ps1 "${{ env.template }}" "${{ steps.remove_quotations.outputs.replaced }}" "${{ env.acceptable_missing_info }}" )" >> $GITHUB_ENV
+      
+      #Remove sections in the issue body like "Dynamo version", "Stack Trace" because won't be used to predict the issue type
+      - name: Clean Issue Body
+        if: env.analysis_response == 'Valid'
+        id: clean-issue-body
+        run: |
+          echo "parsed_issue_body="$(pwsh  .\\.github\\scripts\\issue_body_cleaner.ps1 "${{ steps.remove_quotations.outputs.replaced }}" )"" >> $GITHUB_ENV
+
+      #The IssuesTypePredicter program receives as a parameter a json string with the issue content, then It's creating the json string in this section based in the issue body
+      - name: Create Issue JSON String
+        if: env.analysis_response == 'Valid'
+        id: create-issue-json
+        run: |
+          mkdir IssuesTypePredicter
+          echo "issue_json_string="$(pwsh  .\\.github\\scripts\\get_issue_json_body.ps1 "${{ github.event.issue.number }}" "${{ github.event.issue.title }}" "${{env.parsed_issue_body}}" )""  >> $GITHUB_ENV
+
+      #Now checkout the IssuesTypePredicter source code from the repo https://github.com/DynamoDS/IssuesTypePredicter
+      - name: Checkout IssuesTypePredicter
+        if: env.analysis_response == 'Valid'
+        uses: actions/checkout@v2
+        with:
+          repository: DynamoDS/IssuesTypePredicter
+          path: IssuesTypePredicter
+
+      #Builds the solution IssuesTypePredicter.sln (this contains two VS2019 ML.NET projects)
+      - name: Build Issues Type Predicter 
+        if: env.analysis_response == 'Valid'
+        run: | 
+          dotnet build ./IssuesTypePredicter/IssuesTypePredicter.sln --configuration Release
+          cp ./IssuesTypePredicter/IssuesTypePredicterML.ConsoleApp/bin/Release/netcoreapp3.1/MLModel.zip .
+      
+      #Execute the IssuesTypePredicter program and pass as a parameter the json string (which contains the issue info)
+      - name: Run Issues Type Predicter
+        if: env.analysis_response == 'Valid'
+        run: | 
+          echo "is_wish_list="$(dotnet run -p ./IssuesTypePredicter/IssuesTypePredicterML.ConsoleApp/IssuesTypePredicterML.ConsoleApp.csproj -v q "${{env.issue_json_string}}")"" >> $GITHUB_ENV
+
+      #If the is_wish_list variable contains 1 means that is a wishlist issue and label the issue with the word "Wishlist"
+      - name: Label Wishlist
+        if: contains(env.is_wish_list,'IsWishlist:1') && env.analysis_response == 'Valid'
+        run: |
+          curl -v -u admin:${{ secrets.GITHUB_TOKEN }} -d '{"labels": ["Wishlist"]}' ${{ github.event.issue.url }}/labels
```

Comment:
Hi @RobertGlobant20 other than put label on it, do we plan to move the issue to wishlist repo?

Reply:
Hi @QilongTang 
There is a existing workflow that moves the issues to the corresponding repo based in the label, so if we label a issue like "Wishlist" the "Move issue by labels" workflow will move the issue to the DynamoWishlist repo.

![image](https://user-images.githubusercontent.com/61755417/107239431-78844780-69ee-11eb-877d-605ede020fb5.png)